sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json

```diff
@@ -45,6 +46,9 @@
   "sunpy.timeseries.tests.test_timeseriesbase.test_noaa_txt_pre_peek": "fbda770f164bd38a7741d3393e9f93040b188e928521cc53379ba2b78beb582c",
   "sunpy.timeseries.tests.test_timeseriesbase.test_generic_ts_peek": "73a71e717b7dd0dd345f0f8a8bfa7f06387607417b3ecd123adfd8f630f93d51",
   "sunpy.visualization.animator.tests.test_basefuncanimator.test_lineanimator_figure": "dbf47ab983a844621f912b2f5a2d9ee657d3f661678c16a07afc7217f50e3fd4",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_simple_plot": "0b4d489854b85b6fe469c61c8b1920183e06bb59f2075630d23f46a3c516e030",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_colorbar_buttons": "fcd5d64e48e45c3eebe73a835a22921bfb80b5a8b63eb85faddb68a2cfb447d1",
+  "sunpy.visualization.animator.tests.test_mapsequence.test_map_sequence_animator_wcs_colorbar_buttons_default_labels": "efc609d0263597f1c66ff578262f14a2e1cee73bf90e2fb2f44334d82fed85f6",
```

Comment:
```suggestion
```

Reply:
Sorry this keeps happening, I need to fix this.